docs/specification.md

```diff
@@ -0,0 +1,169 @@
+`delphi.testnet.tezos.serokell.team:8732`
+
+Now we can start to interact with Tezos test net, but we still have no account with money. So, lets 
+get it:
+* Navigate to https://faucet.tzalpha.net and download the file. This file will contains all
+the required information for account activation.
+* Activate account using 
+`tezos-client activate account <account_alias> with <path_to_the_downloaded_file>` command
+* After that you will be able to see your address in the net. Lets open 
+[block explorer](https://tzkt.io/), choose `carthage` net and search for the address.
+
+One more thing you need to do before swap test is to create the account with whom the swap will be 
+produced:
+* Generate new account `tezos-client gen keys bob`
+* And transfer some money to him `tezos-client transfer 1000 from alice to bob`
+* You will see error about burn cap and needed quantity to execute this operation.
+* Add ` --burn-cap <burn_cap_number_from_error message>` to the previous command and try again.
+
+You can easily see all addresses using `tezos-client list known addresses`
+
+#### Originate the contract
+
+After creating identity and accounts we can originate our bridge contract. You can find contract
+code [here](../contract/morley-contract/contract.tz) with 
+[storage](../contract/morley-contract/storage.tz) example. Lets originate this contract and add
+previously created accounts to its storage (you alos may need to add a --burn-cap flag to this 
+command):
+
+```
+tezos-client originate contract blndOnTezos transferring 1 from alice running <path_to_contract> \
+  --init 'Pair (Pair (Pair {Elt <alice_addres> 1000; Elt <bob_address> 1000} { }) \
+  (Pair <alice_addres> (Pair False 2000))) (Pair { } { })'
+```
+
+This command will originate bridge contract with alias `blndOnTezos`, two accounts holding 
+1000 tokens and `alice` as administrator. Also in the command output you can see contract address 
+and find it in the [block explorer](https://better-call.dev/).
+
+Now you are ready to interact with Tezos part of `Bridge` contract.
+
+### Ethereum
+
+#### Build
+
+First clone [Blend](https://github.com/StakerDAO/blend) repository, it contains all needed
+things to build, deploy and interact with Bridge contract. To build it you need to have Node v14 and
+install all with `npm install`. Then build and compile contracts with `npm run build` and 
+`npx oz compile`.
+
+#### Account
+
+One more thing you need to do before contract deploy is Ethereum account with money on testnet.
+To get it create `.env` file and fill it as in `.env.example`. Infura data could be taken from
+[infura](http://infura.io/) service after registration and new project creation. And mnemonic with 
+12/24 length generated [here](https://iancoleman.io/bip39/).
+
+After that run `npx oz accounts`. You will see all accounts with your mnemonic and the last step is
+to ask money [here](https://goerli-faucet.slock.it/) for default account.
```

Comment:
What is default account? It is not very clear for me (and probably for other readers too (or I am just stupid))